src/utilities/codemirror/vml/vmlTreeUtils.ts

```diff
@@ -0,0 +1,91 @@
+import type { SyntaxNode } from '@lezer/common';
```

Comment:
Is it possible to write unit tests for this class?

Reply:
Addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/31ea81bdbeeeac6f4782efc5bdf434d6b48c7a18

I didn't write any playwright tests yet, I'd like to hold off until we decide if this is staying in core, or build language specific modules that will execute playwright tests, otherwise they'd become unused quickly